polkadot.json

```diff
@@ -124,7 +124,7 @@
       {
         "id": 17,
         "tweetReason": "Share the love on Twitter - we'll be rewarding the best ones with swag and prizes! ❤️",
-        "tweetContent": "I'm building on @Polkadot with @easya_app on my #60DaysOfPolkadot!\n\nCurrently learning about... \n\n🧑‍💻 How parachains work!\n⛓ How to build blockchains with @substrate_io \n🖼 How to speed up development 10x",
+        "tweetContent": "I'm building on @Polkadot with @easya_app on my #60DaysOfPolkadot!\n\nCurrently learning about... \n\n🧑‍💻 How parachains work!\n⛓ How to build blockchains with @Polkadot \n🖼 How to speed up development 10x",
```

Comment:
@platonicsocrates is `#60DaysOfPolkadot` still relevant?

Reply:
this will be updated to #360DaysOfPolkadot in the next big push! we can even just updated this now as a teaser